packages/config/src/env.ts

```diff
@@ -5,6 +5,13 @@ type Env = {
     apiKey: string;
   },
   githubToken?: string;
+  emailProvider: {
+    providerMailId?: string,
```

Comment:
_provider mail id_ não faz sentido 😛 :
https://translate.google.com/?sl=en&tl=pt&text=provider%20mail%20id&op=translate

```suggestion
    emailProviderId?: string,
```

Ou (como já está no objeto `emailProvider`):

```suggestion
    providerId?: string,
```